docs/samples/k8s/hello-world/solution.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   metadata:
     deployment.replicas: "#1"
     service.ports: "[{\"name\":\"port9090\",\"port\": 9090}]"
-    service.type: "LoadBalancer"
+    service.type: "NodePort"
```

Comment:
I understand we need to change this to NodePort because minikube cannot create IP for load balancer service without tunnel. We can modify the test cases, but here is the sample we used for maestro. It is better we keep this as LoadBalancer but add "minikube tunnel" at 07.maestroTest. Let's have a sync tomorrow.

Reply:
fixed